src/linear_samplers/linear_sampler_helpers/helpers_cyclic.jl

```diff
@@ -0,0 +1,44 @@
+"""
+    init_blocks_cyclic!(type::Union{LinSysBlkColRandCyclic,LinSysBlkRowRandCyclic}, dim)
+
+This function intializes the `blockSize`, `n_blocks`, and `order` values for the `LinSysBlkColRandCyclic` and `LinSysBlkRowRandCyclic` data structures. If a set of blocks is already defined by the user then it checks it the vector specifying the blocks is the same as n_blocks * BlockSize, if it is not it appends the necessary number of indicies from the second to last block to the final block. If the blocks are not premade, it simply allocates blocks in sequential order.
+
```

Comment:
This follows the same format of comments we have for functions. But I am not sure if it is as descriptive as it ought to be. Something to consider for v0.2.

Reply:
A input, output type description may be more clear in the future.